From 188d325434e417458beb25f7cd1922c7b358a96f Mon Sep 17 00:00:00 2001
From: Eugene Syromiatnikov <esyr@redhat.com>
Date: Thu, 5 Sep 2019 18:48:50 +0200
Subject: [PATCH 30/43] Revert "iwlwifi: mvm: report # of LTF symbols for
 extended range SU PPDUs"

This reverts commit 925bc2b189ec05a2993d3870f4c9c7edaf60d088.
---
 drivers/net/wireless/intel/iwlwifi/mvm/rxmq.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: src/drivers/net/wireless/intel/iwlwifi/mvm/rxmq.c
===================================================================
--- src.orig/drivers/net/wireless/intel/iwlwifi/mvm/rxmq.c	2019-09-06 01:25:45.376293592 +0200
+++ src/drivers/net/wireless/intel/iwlwifi/mvm/rxmq.c	2019-09-06 01:25:45.550291373 +0200
@@ -1236,8 +1236,7 @@
 	he->data5 |= le16_encode_bits(ltf, IEEE80211_RADIOTAP_HE_DATA5_LTF_SIZE);
 
 	switch (he_type) {
-	case RATE_MCS_HE_TYPE_SU:
-	case RATE_MCS_HE_TYPE_EXT_SU: {
+	case RATE_MCS_HE_TYPE_SU: {
 		u16 val;
 
 		/* LTF syms correspond to streams */
@@ -1289,6 +1288,7 @@
 				val));
 		}
 		break;
+	case RATE_MCS_HE_TYPE_EXT_SU:
 	case RATE_MCS_HE_TYPE_TRIG:
 		/* not supported */
 		break;
